docs/build/cmake-presets-vs.md

```diff
@@ -264,6 +264,8 @@ The `architecture.strategy` and `toolset.strategy` values tell CMake how to hand
 
 You should use `set` with IDE generators like the Visual Studio Generator. You should use `external` with command-line generators like Ninja. These values allow vendors like Visual Studio to source the required environment before CMake is invoked. For more information about the architecture and toolset fields, see the [list of Configure Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#configure-preset).
 
+Additionally, if you do not want any environment sourced, you can set architecture.value to "unspecified" and architecture.strategy to "external". However, you can only do this if you have Ninja set as the generator. Not having any environment sourced could be useful in cases where you are using a toolset other than MSVC where you don't need a specific environment in order to build, or you are in a linux/non-Windows environment. 
+
```

Comment:
As a third example in your last paragraph, we can maybe add something about custom toolchain files. E.g. - "...could be useful in cases where you are using a toolset other than MSVC, using a custom toolchain file, or are working in a non-Windows environment." 

If you think it makes sense, perhaps we can explicitly call out embedded scenarios as an instance where customers are using a custom toolchain file.